### ISSUE TITLE
Write helper to get int version of $_COOKIE['PHPSESSID']

### ISSUE THREAD

**Issue by LeoGalleguillos** [LeoGalleguillos/Superglobal#1]
The following code used elsewhere might be useful:

```
$phpSessionIdString = $_COOKIE['PHPSESSID'] ?? null;
$phpSessionIdInt    = hexdec($phpSessionIdString);
```

**Comment by LeoGalleguillos** [comment]
Resolved in commits `git log b996abd391601115859b9159433ebf2b9ff00c63^1..3b9fb3c5660690951d1da7530d39eaf3727f8e5d`

**Comment by LeoGalleguillos** [comment]
This service is actually pretty terrible. We need to upgrade to `php >= 7.1.0` and use native [session functions](https://www.php.net/manual/en/ref.session.php) instead.
